tox.ini

```diff
@@ -54,9 +55,6 @@ whitelist_externals =
   ansibledevel: sh
 
 [testenv:.dev-env]
-#basepython = python3
-basepython = /home/wk/.pyenv/versions/ansible-lint-py3.8.0-pyenv-venv/bin/python3
-#{[testenv]deps}
```

Comment:
Let's not touch this for now. I've been meaning to make `--dev-env` support work but it's currently broken anyway.